src/test/java/org/medibloc/panacea/PanaceaApiRestClientTest.java

```diff
@@ -24,12 +24,19 @@
     public void testGetAccountTestnet() throws PanaceaApiException {
         PanaceaApiRestClient client = PanaceaApiClientFactory.newInstance().newRestClient("http://52.78.196.16:1317");
         Account acc = client.getAccount("panacea1tkat7m78exe89jkx40e3c7rurytu5pukajdamq");
+        assertEquals("panacea1tkat7m78exe89jkx40e3c7rurytu5pukajdamq", acc.getValue().getAddress());
         System.out.println(acc);
-        BlockInfo bi = client.getBlockByHeight(3171824L);
+
+        //TODO: setup proper integration tests, instead of using the magic block height
+        BlockInfo bi = client.getBlockByHeight(7710L);
+        assertEquals(new Long(7710), bi.getBlockMeta().getHeader().getHeight());
```

Comment:
This test file may not work sometimes, because it communicates with the real Panacea Core.

I'm gonna setup the proper integration test with docker soon: https://github.com/medibloc/medibloc-panacea-api/issues/12.